adminer-editor/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM alpine:3.20
 
 LABEL maintainer="Milan Sulc <sulcmil@gmail.com>"
 
-ENV ADMINER_EDITOR_VERSION=5.3.0
```

Comment:
I think it's a typo, it should be 5.4.1 instead of 5.1.0, right?

Reply:
thanks!